zinc-compile/src/test/scala/inc/DocSpec.scala

```diff
@@ -16,7 +16,7 @@ class DocSpec extends UnitSpec {
     IO.withTemporaryDirectory { cacheDir =>
       IO.withTemporaryDirectory { out =>
         val javadoc = Doc.cachedJavadoc("Foo", cacheDir, local)
-        javadoc.run(List(knownSampleGoodFile), Nil, out, Nil, log, reporter)
+        javadoc.run(List(knownSampleGoodFile), Nil, out, Nil, null, log, reporter)
```

Comment:
One option I considered is to have a `noop` `ClassFileManager` to pass around when no work is to be done.

Two options to do this: 
* in `ClassfileManager` similar to the existing predefined `ClassFileManager`:`deleteImmediately`
* As a default option in `compiler-interface` along with `ClassFileManager` is defined

I slightly prefer the former, just to keep the interface _interface_, that will introduce a dependency of `zincCore` to `zinc-compile`, is that ok?